syntax = "proto3";

message TradeResponse {
    int64 isSuccess = 1;
}